core/shared/src/main/scala/zio/prelude/NewtypeModule.scala

```diff
@@ -176,6 +176,14 @@ private[prelude] sealed trait NewtypeModule {
   sealed trait Newtype[A] extends NewtypeVersionSpecific[A] {
     type Type
 
+    /**
+     * Derives an instance of a type class for the new type given an instance
+     * of the type class for the underlying type. The caller is responsible for
+     * the type class being a valid instance for the new type.
+     */
+    protected def derive[TypeClass[_]](ev: TypeClass[A]): TypeClass[Type] =
```

Comment:
Shouldn't this be `implicit ev: TypeClass[A]` to leverage the compiler for summoning the implicit instance?

Reply:
Yes! Was playing with the signature and accidentally deleted the most important part!